### DISCUSSION TITLE
Ask about KitchenSink app.

### DISCUSSION THREAD

**Discussion by james-ep01** [clientIO/joint#2829] [Q&A]
### Introduction

I have a question regarding the KitchenSink app.

The KitchenSink app seems a little strange in the new version 4.1.
I did the basic installation and execution.

npm install
npm start

While upgrading, I discovered two strange things. 

first .
In the KitchenSink app, the halo type pie does not work. 
In my app, halo is pie. My app works after upgrading to v4.1, 
but pie does not work in KitchenSink. Nothing comes out. 

second.
To use the KitchenSink light skin, I would like to refer to joint-light-theme.css created after installation, and the css file has 1361 lines. There is a lot of overlapping CSS, so it is difficult to refer to the light skin.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
1. The pie does not work because the `light` theme does not support the `pie` type.
You can set the theme per plugin, too.
```ts
new ui.Halo({ theme: 'material', ... });
```

2. I am not sure what is the question.

**Comment by james-ep01** [comment]
[joint-light-theme.zip](https://github.com/user-attachments/files/18095867/joint-light-theme.zip)

This is the installed joint-light-theme.css file.
There are a lot of duplicated css. Do you need a css file with so many repeated classes to handle the light theme? I chose to develop with joint+ because the reference apps are attractive and I think it will greatly shorten my app development time. 

The css file above looks strange to me. Am I the only one who finds it strange? Then it's my problem.

I would be grateful if only the API and examples were abundant.
I chose joint+ for my app development. And I do not make up stories that are not talked about on joint+'s github only for things necessary for my app development.

The development process is 50% complete. It is scheduled to open next month. I will disclose the URL.

Thank you.

**Reply by kumilingus** [reply]
This is a build file. Made from `sass` files. We need to review the build command and remove the duplicities.

In the meantime, you should be able to utilize all the original CSS in the `joint-light-theme` folder.

**Reply by MartinKanera** [reply]
Thank you for your feedback! We appreciate you sharing your thoughts on joint-light-theme. We’ve have made improvements to the SCSS preprocessing, which as a result will leave us with a generated file free of any duplicates. We’re currently finalizing the updates, and a new version will be released soon.
